resources/data/label/register.get.json

```diff
@@ -0,0 +1,22 @@
+[
+    {
```

Comment:
Can you brief about the use and purpose of these labels ?

Reply:
@ankurkakroo2 
 making the labels dynamic , so that we can change anything easily if we require.
Let me know should I change?